src/renderer/views/SubscribedChannels/SubscribedChannels.js

```diff
@@ -162,16 +162,20 @@ export default defineComponent({
 
     thumbnailURL: function(originalURL) {
       let newURL = originalURL
-      const hostname = new URL(originalURL).hostname
+      // Sometimes relative protocol URLs are passed in
+      if (originalURL.startsWith('//')) {
```

Comment:
Imo the whole thumbnailURL method can be removed as we already do invidious replacing through a function in api/invidious, we just need another function to fix up the local api urls (add missing https). Also if we save invidious links to the db I believe they will get overwritten by other parts of the app.

Reply:
Tldr: this method rewrites your entire subscription file if you change to invidious/change invidious instances and we should probably only use the youtube link

Reply:
Make another PR I guess ~_~